### PR TITLE
Feature/material bootstrap

### PR DESCRIPTION
**Pull request by elnzv**

### CODE REVIEW

**Comment by elnzv** [comment]
Hey @apinf/developers review this pull request.

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .meteor/local
 .meteor/meteorite
 .meteor/.id
+/config.json
```
Comment (brylie) [inline .gitignore]: We have not decided on 'config.json' for team configuration files. I am still using 'settings.json' in my local development process. We need to talk about what configuration file name to use going forward.

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 settings.json
 .idea/
-settings.json
```
Comment (brylie) [inline .gitignore]: Removing this line without notice may allow team members to inadvertently commit their private settings.

File: client/style/js/ripples.js [hunk 3]
```diff
@@ -0,0 +1,324 @@
+/* Copyright 2014+, Federico Zivolo, LICENSE at https://github.com/FezVrasta/bootstrap-material-design/blob/master/LICENSE.md */
```
Comment (brylie) [inline client/style/js/ripples.js]: It is becoming common to put 3rd party libraries, such as this, in *client/compatibility*

http://joshowens.me/how-to-organize-your-meteor-js-app/

File: client/style/js/material.js [hunk 4]
```diff
@@ -0,0 +1,226 @@
+/* globals jQuery */
```
Comment (brylie) [inline client/style/js/material.js]: Consider putting this in *client/compatibility*

File: .gitignore (hunk 2 above)
Comment (frenchbread) [inline .gitignore]: Agreed with Brylie at this point, but as additional solution settings.json file can be stored somewhere else, for example on desktop. And is this case launching of the dev server would be:
```
meteor --settings /Users/user/Desktop/settings.json
```
This is just an example. I don't mind keeping settings.json file in .gitignore

Comment (elnzv) [inline .gitignore]: Yep, that was a duplication.

**Pull request merged**